packages/headless/src/app/engine.ts

```diff
@@ -199,15 +197,6 @@ function getUpdateAnalyticsConfigurationPayload(
     apiBaseUrl,
   };
 
-  // TODO KIT-2844
-  if (payloadWithURL.analyticsMode !== 'next' && doNotTrack()) {
-    logger.info('Analytics disabled since doNotTrack is active.');
-    return {
-      ...payloadWithURL,
-      enabled: false,
-    };
-  }
-
```

Comment:
The ticket KIT-2844 is in the v4 epic so we might have to keep this. https://coveord.atlassian.net/browse/KIT-2844